tests/steamship_tests/client/test_client.py

```diff
@@ -16,3 +19,56 @@ def test_user():
     user = User.current(client).data
     assert user.id is not None
     assert user.handle is not None
+
+
+def test_client_creation_with_new_autogen_space():
+    client1 = get_steamship_client(create_space=True)
+    client2 = get_steamship_client()
+    client3 = get_steamship_client()
+
+    assert client2.config.space_id is None
```

Comment:
Wouldn't it be nicer if space_id and space_handle were always populated, even if the default space is used?

Reply:
It's an extra round trip to the server, but I'm not against it!